larrecodnn/ImagePatternAlgs/Tensorflow/Modules/EmTrack.h

```diff
@@ -0,0 +1,358 @@
+#ifndef EMTRACK_H
```

Comment:
Only comment for this file is that the spacing is inconsistent.  I think we've tried using `clang-format` on this file before, and it failed.  Perhaps it would work to use `astyle`?